src/org/omegat/gui/main/MainWindowMenuHandler.java

```diff
@@ -324,7 +324,7 @@ public void projectAccessCurrentSourceDocumentMenuItemActionPerformed(int modifi
             return;
         }
         File toOpen = new File(root, path);
-        if ((modifier & Java8Compat.getMenuShortcutKeyMaskEx()) != 0) {
```

Comment:
Here is a bug that we compare with `KeyEvent.META_DOWN_KEY` or `KeyEvent.CTRL_DOWN_KEY` which is returned by `Java8Compat.getMenuShortcutKeyMaskEx()` but caller call here with `ActionEvent.CTRL_KEY` or `ActionEvent.META_KEY`. It is a reason that a modifier has never been worked.

Reply:
The bug is introduced in v3.5.1 at [a first commit](https://github.com/omegat-org/omegat/commit/fab16c8b1d11ce49d872aab86c7b52ad47267e4b) when adding the feature. At that time, `KeyEvent.META_KEY` and `ActionEvent.META_KEY`  has same value, then it works like as expected.

in v4.2.0, when supporting Java 11 runtime [a change](https://github.com/omegat-org/omegat/commit/98127c352a2b1bb63f1a831389aaa7fc79a83966) breaks the feature. Then compiling with Java11 it stop working.

Reply:
Interesting that nobody noticed that, even though the feature was discussed in 4.1. Thank you for checking the history of the bug.